.github/workflows/clp-core-build.yaml

```diff
@@ -0,0 +1,48 @@
+name: Test Building CLP Core on Ubuntu Focal, Bionic and Centos 7.4
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'components/core/**'
+      - '.github/workflows/clp-core-build.yaml'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'components/core/**'
+      - '.github/workflows/clp-core-build.yaml'
+  workflow_dispatch:
+
+jobs:
+  build-focal:
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-focal:main
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build CLP Core
+        uses: ./.github/actions/clp-core-build
+
+  build-bionic:
+    runs-on: ubuntu-20.04
+    container:
+      image: ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-bionic:main
```

Comment:
If you want, you could rewrite this in one line as `container: ghcr.io/y-scope/clp/clp-core-dependencies-x86-ubuntu-bionic:main`